addons/stock_account/models/account_invoice.py

```diff
@@ -44,6 +44,10 @@ def _get_last_step_stock_moves(self):
         """
         return self.env['stock.move']
 
+    def _get_invoice_snln_values(self):
+        """ Will be overriden by sale_stock in order to show the SN/LN on the invoices """
+        return []
```

Comment:
why not everything in sale_stock? it won't work without a so anyway since we use it to link stock moves and inv lines